OmarManassra/task_manager.py

```diff
@@ -0,0 +1,93 @@
+from  utils import validate_date,get_valid_input,validate_status,validate_priority
+from task import Task,UrgentTask
+# List contains all the tasks 
+tasks_list=[]
```

Comment:
why did you use a list to store your value, why not use a dictionary with the key task if and value the task itself?